src/styles/main.scss

```
@@ -0,0 +1,2 @@
+@import "variables.scss";
+@import "stopwatch.scss";
```

Можна вказувати просто без розширень 
```suggestion
@import "variables";
@import "stopwatch";
```